livingatlas/pipelines/src/main/java/au/org/ala/pipelines/beam/ALAVerbatimToInterpretedPipeline.java

```diff
@@ -103,7 +101,8 @@ public class ALAVerbatimToInterpretedPipeline {
 
   public static void main(String[] args) throws IOException {
     VersionInfo.print();
-    String[] combinedArgs = new CombinedYamlConfiguration(args).toArgs("general", "interpret");
+    String[] combinedArgs =
```

Comment:
can remove this change ? looks like its only formatting. Suggest rolling it back

Reply:
defaultDateFormat is added in